frontends/api/src/mitxonline/test-utils/factories/courses.ts

```diff
@@ -122,7 +123,9 @@ const course: PartialFactory<CourseWithCourseRunsSerializerV2> = (
     Array.from({ length: faker.number.int({ min: 1, max: 3 }) }).map(() =>
       courseRun(),
     )
-  const nextRunId = overrides.next_run_id ?? faker.helpers.arrayElement(runs).id
+  const nextRunId = has(overrides, "next_run_id")
```

Comment:
With the original version of this factory, it was impossible to specify null `next_run_id`, even thought that is possible via the api.